### ISSUE TITLE
Various simple lore errors exist in twisted documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1829]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1829|
|Type|defect|
|Created|2006-06-17 21:00:40Z|

The following currently shows up in process-docs output:

```
WARNING: doc/core/upgrades/index.xhtml:2:4: title and h1 text differ
WARNING: doc/core/upgrades/index.xhtml:invalid format
WARNING: Walker failures
	HTML
WARNING: doc/core/upgrades/2.0/components.xhtml:9:4: title and h1 text differ
WARNING: doc/core/upgrades/2.0/index.xhtml:1:43: not exactly one h1
WARNING: doc/core/upgrades/2.0/split.xhtml:48:3: contains quote
WARNING: doc/core/upgrades/2.0/components.xhtml:invalid format
WARNING: doc/core/upgrades/2.0/index.xhtml:invalid format
WARNING: doc/core/upgrades/2.0/split.xhtml:invalid format
WARNING: Walker failures
```

<details><summary>Searchable metadata</summary>

```
trac-id__1829 1829
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1150578040000000 1150578040000000
changetime__1150587818000000 1150587818000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

All these warnings fixed in [branches/process-docs-1829,](branches/process-docs-1829,) please review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun|
|-|-|

```
[22:43] <foom2> exarkun: looks reasonable I think.
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17268]) Merge process-docs-1829

Author: exarkun
Reviewer: foom
Fixes [#1829](https://github.com/twisted/twisted/issues/1829)

Correct several lore markup errors and update some content which was out
of date.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
